images/scaletesting/README.md

```diff
@@ -0,0 +1,96 @@
+# Scale Testing for Kafka
+
+This image provides an easy way to run `kafka-producer-perf-test` and `kafka-consumer-perf-test`  scripts.
+
+Example deployment:
```

Comment:
Should we refer to the example deployment in the examples folder instead of copying its contents here?

Reply:
linked to the deployment yaml